tests/publiccloud/ssh_interactive_init.pm

```diff
@@ -21,13 +21,8 @@ sub run {
     my ($self, $args) = @_;
 
     # If someone schedules a publiccloud run with a custom SCHEDULE this causes
-    # the test to break, because we need to pass $args via the main_common.pm.
-    # This check warns about this common pitfall when doing verification runs
-    if (!defined $args) {
-        my $msg = 'Note: Running publiccloud runs with a custom SCHEDULE is not yet supported';
-        record_info('ERROR', $msg, result => 'fail');
-        die($msg);
-    }
+    # the test to break, because we need to pass $args This check warns about this pitfall.
```

Comment:
```suggestion
    # the test to break, because we need to pass $args , so dying earlier and with clear message about root cause
```

Reply:
well I not insist on my version , but I think calling die function as "warning" is too much :D

Reply:
I agree with you. Let's wait for @grisu48